From 0eab509963324da00e69048e8fd669752f6f44e3 Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@xxxxx.com>
Date: Mon, 2 Oct 2023 18:34:17 -0400
Subject: [PATCH] arch: arm64: dts: allwinner: sun50i-h6-cpu-opp: turbo-mode

The H6 runs hot out of the gate so lets down clock it by default

Signed-off-by: Patrick Yavitz <pyavitz@xxxxx.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-cpu-opp.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-cpu-opp.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6-cpu-opp.dtsi
index 0baf0f8e4d27..671bb7ae9bbd 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-cpu-opp.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-cpu-opp.dtsi
@@ -87,6 +87,7 @@ opp-1704000000 {
 			opp-microvolt-speed0 = <1120000 1120000 1200000>;
 			opp-microvolt-speed1 = <1060000 1060000 1200000>;
 			opp-microvolt-speed2 = <1060000 1060000 1200000>;
+			turbo-mode;
 		};
 
 		opp-1800000000 {
@@ -96,6 +97,7 @@ opp-1800000000 {
 			opp-microvolt-speed0 = <1160000 1160000 1200000>;
 			opp-microvolt-speed1 = <1100000 1100000 1200000>;
 			opp-microvolt-speed2 = <1100000 1100000 1200000>;
+			turbo-mode;
 		};
 	};
 };
-- 
2.39.2

